source/content/php-versions.md

```diff
@@ -24,7 +24,7 @@ Changes made to the `pantheon.yml` file on a branch **are not** detected when cr
 ### Available PHP Versions
 The recommended PHP versions available on Pantheon are:
 
-- [7.4](https://v74-php-info.pantheonsite.io/)
+- [7.4](https://v74-php-info.pantheonsite.io/) is available for sites on [COE](/platform-considerations#compute-optimized-environments-coe). Other users may experience errors uploading images.
```

Comment:
The extra text is fine, but a little long for inlining in the bullet list. Could this be a footnote or callout?

Reply:
I get what you're saying, and I think we can say it in fewer words. I do prefer keeping the disclaimer as close to the potential issue as possible and I'm passing along to copy review for decision.

Reply:
Also, a technical nit: there really isn't any problem with uploading images. jpg images upload fine, and if you display the original image, it will show up just fine. The reason that uploaded images appears to be problematic is that Drupal resizes the image for display, and the resize operation is what is failing.

If you want to say it in **fewer** words, though, then maybe "only supported on COE" would work.

Reply:
Maybe "only supported on COE" could link to a separate page with an expanded explanation? Maybe add a section to https://pantheon.io/docs/platform-considerations that explains that GD + JPG only works on COE?